storybook/src/Storybook/BiometricsPopup.qml

```diff
@@ -15,21 +15,23 @@ Dialog {
 
     required property string password
     required property string pin
-    required property string selectedProfileIsKeycard
-
-    // password signals
-    signal accountLoginError(string error, bool wrongPassword)
+    required property bool selectedProfileIsKeycard
 
     // biometrics signals
     signal obtainingPasswordSuccess(string password)
-    signal obtainingPasswordError(string errorDescription, string errorType /* Constants.keychain.errorType.* */, bool wrongFingerprint)
```

Comment:
Wondering why you removed the "failed" state/signal here? If the fingerprint is not recognized by the OS, what happens?

Reply:
On mac there is no way to fail, only cancellation is possible. When fingerprint is not recognized, user is asked to try again.

Reply:
Hmm, really no way to something like exceed the number of "failed" fingerprint attempts? @igor-sirotin

Reply:
There is a way to exceed the number of attempts for the TouchID.
But then OS allows to use the OS password to access Keychain. And password has endless attempts.